activiti-services/activiti-services-events/src/main/java/org/activiti/services/events/MessageProducerActivitiEventListener.java

```diff
@@ -24,10 +29,34 @@ public MessageProducerActivitiEventListener(ProcessEngineChannels producer,
 
     @Override
     public void onEvent(ActivitiEvent event) {
+        CommandContext currentCommandContext = Context.getCommandContext();
```

Comment:
Sorry, I forget to mention, but we'll try to avoid using PowerMock. What I had in mind initially was to encapsulate the `Context.getCommandContext` inside a wrapper which can be easily mocked. For the moment let's keep it in the "old" style. That's fine.